src/parser.js

```diff
@@ -218,6 +219,31 @@ function runInSandbox({ markup, query, cacheId }) {
   sandbox.ensureMarkup(markup);
 
   const result = sandbox.eval(query);
+  result.markup = markup;
+  result.query = query;
+
+  result.elements = ensureArray(result.data)
+    .filter((x) => x?.nodeType === Node.ELEMENT_NODE)
+    .map((element) => {
+      const { suggestion, data } = getQueryAdvise({
+        rootNode: sandbox.rootNode,
+        element,
+      });
+
+      const cssPathString = cssPath(element, true).toString();
+      const target = sandbox.rootNode.querySelector(
+        cssPathString.replace(/^body > /, ''),
+      );
+      return {
+        suggestion,
+        data,
+        target: target,
+        cssPath: cssPathString,
+        queries: getAllPossibileQueries(target),
+      };
+    });
+
+  result.accessibleRoles = getRoles(sandbox.rootNode);
```

Comment:
This looks like a copy from line 113:129. Can't we add the `queries: getAllPossibileQueries(target)` there, instead of overwriting the `elements` collection here?

https://github.com/testing-library/testing-playground/blob/d75544b4db24f2e0cbb778bfc6126f16dc9ccc72/src/parser.js#L113-L129

Reply:
Also, now we return `queries`, are `suggestion` and `data` still being used?

Reply:
I see that there were a lot of duplicate codes. Now I think that I cleaned everything. 
`suggestion` is still using in `ResultSuggestion`, it is the first suggestion shown to the user and I think we can't drop for now.
`data` maybe can be dropped but I need to look better

Reply:
I think that the drop of this props can be with the others `level` and `target`. I will open a new PR for them